### PR TITLE
With antsibull-changelog >= 0.14.0 and the sidecar docs feature, test and filter plugins are detected if they have version_added

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Assuming you use the devel branch, milestone branch, or ansible-core >= 2.14 once released :)

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/community/collection_development_process.rst
docs/docsite/rst/community/development_process.rst

### CODE REVIEW

**Comment by felixfontein** [comment]
Ref: https://github.com/ansible-community/antsibull-changelog/pull/79

File: docs/docsite/rst/community/collection_development_process.rst [hunk 1]
```diff
@@ -61,8 +61,7 @@ More precisely:
...
 * Every feature PR must have a changelog fragment.
-* New modules and plugins (except jinja2 filter and test plugins) must have ``versions_added`` set correctly, and do not need a changelog fragment. The tooling detects new modules and plugins by their ``versions_added`` value and announces them in the next release's changelog automatically.
-* New jinja2 filter and test plugins, and also new roles and playbooks (for collections) must have a changelog fragment. See :ref:`changelogs_how_to_format_j2_roles_playbooks` or the `antsibull-changelog documentation for such changelog fragments <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst#adding-new-roles-playbooks-test-and-filter-plugins>`_ for information on what the fragments should look like.
+* New modules and plugins (including jinja2 filter and test plugins) must have ``versions_added`` set correctly in their documenation, and do not need a changelog fragment. The tooling detects new modules and plugins by their ``versions_added`` value and announces them in the next release's changelog automatically.
```
Comment (briantist) [inline docs/docsite/rst/community/collection_development_process.rst]:
```suggestion
* New modules and plugins (including jinja2 filter and test plugins) must have ``version_added`` entries set correctly in their documentation, and do not need a changelog fragment. The tooling detects new modules and plugins by their ``version_added`` values and announces them in the next release's changelog automatically.
```

File: docs/docsite/rst/community/development_process.rst [hunk 2]
```diff
@@ -161,8 +161,7 @@ More precisely:
...
 * Every feature PR must have a changelog fragment.
-* New modules and plugins (except jinja2 filter and test plugins) must have ``versions_added`` set correctly, and do not need a changelog fragment. The tooling detects new modules and plugins by their ``versions_added`` value and announces them in the next release's changelog automatically.
-* New jinja2 filter and test plugins, and also new roles and playbooks (for collections) must have a changelog fragment. See :ref:`changelogs_how_to_format_j2_roles_playbooks` or the `antsibull-changelog documentation for such changelog fragments <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst#adding-new-roles-playbooks-test-and-filter-plugins>`_ for information on what the fragments should look like.
+* New modules and plugins (including jinja2 filter and test plugins) must have ``versions_added`` set correctly in their documentation, and do not need a changelog fragment. The tooling detects new modules and plugins by their ``versions_added`` value and announces them in the next release's changelog automatically.
```
Comment (briantist) [inline docs/docsite/rst/community/development_process.rst]:
```suggestion
* New modules and plugins (including jinja2 filter and test plugins) must have ``version_added`` entries set correctly in their documentation, and do not need a changelog fragment. The tooling detects new modules and plugins by their ``version_added`` values and announces them in the next release's changelog automatically.
```

File: docs/docsite/rst/community/development_process.rst [hunk 3]
```diff
@@ -300,44 +299,15 @@ You can find more example changelog fragments in the `changelog directory <https
 
 After you have written the changelog fragment for your PR, commit the file and include it with the pull request.
 
-.. _changelogs_how_to_format_j2_roles_playbooks:
+.. _changelogs_how_to_format_playbooks:
```
Comment (briantist) [inline docs/docsite/rst/community/development_process.rst:302]: was a search done by any chance to see if there are any referenced to the old slug?

Comment (felixfontein) [reply docs/docsite/rst/community/development_process.rst:302]: It was only referenced from the two other paragraphs I changed above.

**Review by briantist — APPROVED** [review]
lgtm!

**Comment by felixfontein** [comment]
@sivel you wanted to be informed on PRs that affect the changelog process. Here's one :)

**Review by samccann — APPROVED** [review]

**Comment by felixfontein** [comment]
@briantist @samccann thanks for reviewing and merging!

**Pull request merged**